src/models/clients.models.ts

```diff
@@ -7,24 +7,29 @@ export function fetchClients() {
 }
 
 export function getClient(data) {
-  const gov_id = [data.gov_id];
+<<<<<<< Updated upstream
+  console.log("data", data);
   return db
-    .query("SELECT * FROM clients WHERE gov_id = $1", gov_id)
+    .query("SELECT * FROM clients WHERE gov_id = $1", [data])
+=======
+  return db
+    .query("SELECT * FROM clients WHERE gov_id = $1", data)
+>>>>>>> Stashed changes
     .then((client) => {
-      return client.rows;
+      return client.rows[0];
     });
 }
 
 export function attachSurveysToClient(protocolId, clientId) {}
 
 export async function createTreatment(data) {
-  const govId = await addClient(data.client);
-  const treatment = [govId, client.protocolId, client.startDate];
-  return db.query(
-    `INSERT INTO treatment (client_id,protocol_id,start_date) 
-    VALUES ($1,$2,$3)`,
-    treatment
-  );
+  // const govId = await addClient(data.client);
+  // const treatment = [govId, client.protocolId, client.startDate];
+  // return db.query(
+  //   `INSERT INTO treatment (client_id,protocol_id,start_date)
+  //   VALUES ($1,$2,$3)`,
+  //   treatment
+  // );
```

Comment:
If you are not using this code then remove it

Reply:
this is george's